lib/search-summary/header/index.js

```diff
@@ -1,9 +1,10 @@
 import React, { Component, PropTypes } from 'react';
 import './style.css';
-const postImage = require('../../../src/assets/goingplaces.svg');
+var Scroll = require('react-scroll');
+var scroll = Scroll.animateScroll;
```

Comment:
why `require` instead of import ?
I'd suggest to change to this:
```js
import { animateScroll as scroll } from 'react-scroll';
```
unless there is any reason I can't see